ci-operator/config/openshift/file-integrity-operator/openshift-file-integrity-operator-master.yaml

```diff
@@ -29,7 +29,7 @@ operator:
     context_dir: .
     dockerfile_path: bundle.Dockerfile
   substitutions:
-  - pullspec: quay.io/file-integrity-operator/file-integrity-operator:1.3.0
+  - pullspec: quay.io/file-integrity-operator/file-integrity-operator:1.3.3
```

Comment:
@rhmdnd What if we had substitutions for the current tag `1.3.3` and the future one `1.3.4`?
This way it will be a bit more future proof.

There can be more than on substitutions:
https://docs.ci.openshift.org/docs/how-tos/testing-operator-sdk-operators/#building-operator-bundles

Reply:
It is not clear to me in which order the bundle update in CI happens compared to the release PR. Which one is merged first?

It seems to me that this one comes first, as it has substitutions to make the release PR work.

Reply:
Correct - we have to merge this one first, so the bundle upgrade will work in the release PR. Then when we land the release PR, I expect we'll need to come back an bump this so that it's using 1.3.4 (the latest release).